akka-remote/src/main/scala/akka/remote/MessageSerializer.scala

```diff
@@ -53,27 +54,28 @@ private[akka] object MessageSerializer {
 
     // FIXME: This should be a FQCN instead
     headerBuilder.serializer = serializer.identifier.toString
```

Comment:
OK, we can keep the numbering scheme, or alternatively, give names to the serializers.